styles.css

```diff
@@ -500,4 +505,17 @@ body {
         font-size: 24px;
     }
 
+    .speaker-img {
+        position: relative;
+        top: -80px;
+        margin-right: 12px;
+        height: auto;
+        width: 100%;
+    }
```

Comment:
- [ ] Great job creating the speakers' section dynamically from JavaScript 💯 Kindly add some more margin to each speaker div in the mobile view so that they do not touch the next speaker div 👍 
- [ ] Also, kindly implement the functionality of the `More` button in the mobile view. When a user clicks the `More` button all the remaining speakers should be displayed in the mobile view. It would be great if you could implement this functionality 😄 
- Your design:
![Annotation 2022-04-23 070223](https://user-images.githubusercontent.com/85113567/164882271-cf3fa5bd-1b29-41b4-ba40-2ab44d69c4b0.jpg)
- Required design:
![Annotation 2022-04-23 064937](https://user-images.githubusercontent.com/85113567/164882276-7c838552-00b0-497f-a5b1-390eb638b150.jpg)

Reply:
All the changes requested here have been implemented, thanks.